streaming/pom.xml

```diff
@@ -126,6 +126,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
         <configuration>
+          <shadedArtifactAttached>true</shadedArtifactAttached>
```

Comment:
Actually, this will infect 'attachedArtifacts' in the log i supply。
I will remove this in case misleading.